iam/api-client/src/main/java/iam/snippets/GetServiceAccountPolicy.java

```diff
@@ -36,15 +35,17 @@ public static void main(String[] args) throws IOException {
 
   // Gets a service account's IAM policy.
   public static Policy getPolicy(String projectId, String serviceAccount) throws IOException {
+
     //Construct the service account email.
     //You can modify the ".iam.gserviceaccount.com" to match the service account name in which
     //you want to delete the key.
     //See, https://cloud.google.com/iam/docs/creating-managing-service-account-keys?hl=en#deleting
```

Comment:
```suggestion
    // See, https://cloud.google.com/iam/docs/creating-managing-service-account-keys?hl=en#deleting
```

Reply:
I meant, leading space in each line. Please make this change wherever applicable in other snippets as well.